modules/administration_manual/pages/configuration/files/external_storage/smb.adoc

```diff
@@ -3,9 +3,18 @@
 :libsmbclientphp-url: https://github.com/eduardok/libsmbclient-php
 :smbclient-url: https://www.samba.org/samba/docs/man/manpages-3/smbclient.1.html
 :smbcifs-url: https://docs.microsoft.com/en-us/windows/desktop/FileIO/microsoft-smb-protocol-and-cifs-protocol-overview
+:samba-478-url: https://www.samba.org/samba/history/samba-4.7.8.html
+:samba-481-url: https://www.samba.org/samba/history/samba-4.8.1.html 
+:smb2-url: https://en.wikipedia.org/wiki/Server_Message_Block#SMB_2.0
 
 ownCloud can connect to Windows file servers, and other SMB-compatible (Samba) servers, by using the {smbcifs-url}[SMB/CIFS] backend.
 
+[IMPORTANT]
+====
+ownCloud requires at least {samba-478-url}[Samba 4.7.8] or {samba-481-url}[Samba 4.8.1] on the ownCloud server, when the Windows Network Drive Listener is used and the remote Windows/Samba file server requires at least {smb2-url}[SMB 2.0].
+The Windows Network Drive Listener only supports version 1 of the SMB protocol with earlier Samba versions. 
+====
```

Comment:
This sentence sounds for me the following way - maybe I am wrong:
For the owncloud instance we need samba version greater than 4.7.x, for the target host SMB version must be minimum SMB2.0. WND listener on the other hand only supports SMB1. So host must be higher than listener - hmmm.

Reply:
I can't vouch for the differing versions of the listener, Samba, and the SMB protocol, but your text gave me reason to have a think. I like it mostly as it is, but I agree it can be updated so that it's clearer. I'll do that momentarily, and would appreciate your, further, feedback.

Reply:
Change made.

Reply:
That sounds strange because first you require a minimum samba version but then you exclude wnd listener in the xref line from it (with earlier Samba versions). Proposal: we need to contact a knowledged colleague to clarify that, else it will be a pain/guess only...

Reply:
Agreed. Just sent a message to the relevant channel.

Reply:
Any update ?

Reply:
Nothing yet.